From 27488975a61858bbaf72047838074389566e2136 Mon Sep 17 00:00:00 2001
From: Gaetan Nadon <memsize@videotron.ca>
Date: Fri, 13 Feb 2015 21:43:37 -0500
Subject: [PATCH 4/8] Remove left over $(launchagents_DATA) in CLEANFILES

This was left over when reorganizing layout of launchd sources
in commit 567f59d3f8189b92bc46e2af1260f9340f462bdb

Signed-off-by: Gaetan Nadon <memsize@videotron.ca>
Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 3867bea..c1eb5a9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -58,7 +58,7 @@ CPP_FILES_FLAGS = \
 xinitrc_DATA = xinitrc
 
 MAINTAINERCLEANFILES = ChangeLog INSTALL
-CLEANFILES = xinitrc startx $(launchagents_DATA)
+CLEANFILES = xinitrc startx
 
 EXTRA_DIST = xinitrc.cpp startx.cpp \
 		autogen.sh
-- 
1.9.0

